src/Compiler/RefC/RefC.idr

```diff
@@ -921,7 +921,7 @@ createCFunctions n (MkAForeign ccs fargs ret) = do
           let isStandardFFI = Prelude.elem lang ["RefC", "C"]
           let fctName = if isStandardFFI
                            then UN $ Basic $ fctForeignName
-                           else UN $ Basic $ lang ++ "_" ++ fctForeignName
+                           else NS (mkNamespace lang) n
```

Comment:
It would, but as this is outside the `Namespace` module, I feel we shouldn't use `unsafe` `Namespace` functions.

Also, arguably, this would also allow us to use namespaces in language identifiers. eg. `scheme.chez`, `scheme.racket`. Though I'm not sure how useful that would be.